aerleon/lib/nokiasrl.py

```diff
@@ -124,12 +187,12 @@ def _BuildTokens(self) -> Tuple[Set[str], Dict[str, Set[str]]]:
 
         supported_sub_tokens['action'] = {'accept', 'deny'}  # excludes 'reject'
         supported_sub_tokens['option'] = {
-            #  'established',
+            'established',
             'first-fragment',
             'is-fragment',
             'fragments',
             #  'sample',
```

Comment:
Are the commented out sub tokens to be added later? If not can we remove those?

Reply:
leaving as-is for now